app/controllers/oaisys/pmh_controller.rb

```diff
@@ -18,6 +18,12 @@ def bad_verb; end
 
   def identify
     expect_no_args for_verb: :Identify
+
+    respond_to do |format|
+      format.xml do
```

Comment:
tiny tiny tiny nit, but with a one line block, especially one this short,  `format.xml { render :identify }` is slightly more idiomatic